Test/SearchTaskTest.cs

```diff
@@ -87,5 +87,80 @@ public static void SemiSpecificTest()
             var output = File.ReadAllLines(outputPath);
             Assert.That(output.Length == 9);
         }
+
```

Comment:
write summary comment of purpose of unit test and what it's testing (broad overview)